src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```diff
@@ -301,11 +301,9 @@ private void ValidateAndAddTriggerBasedProcessDumpParameters(XmlElement collectD
             {
                 case XmlAttribute attribute when string.Equals(attribute.Name, Constants.CollectDumpAlwaysKey, StringComparison.OrdinalIgnoreCase):
 
-                    if (string.Equals(attribute.Value, Constants.TrueConfigurationValue, StringComparison.OrdinalIgnoreCase) || string.Equals(attribute.Value, Constants.FalseConfigurationValue, StringComparison.OrdinalIgnoreCase))
-                    {
-                        bool.TryParse(attribute.Value, out _collectDumpAlways);
-                    }
-                    else
+                    if ((!string.Equals(attribute.Value, Constants.TrueConfigurationValue, StringComparison.OrdinalIgnoreCase)
+                        && !string.Equals(attribute.Value, Constants.FalseConfigurationValue, StringComparison.OrdinalIgnoreCase))
+                        || !bool.TryParse(attribute.Value, out _collectDumpAlways))
```

Comment:
I have changed a bit the logic to also log the warning about the parameter in case of a failure in the `TryParse` although I don't really expect any problem as we just tested before that the string was equal to `true` or `false`.